src/Monitor/Monitor/MonitorCmdletBase.cs

```diff
@@ -89,8 +89,12 @@ public override void ExecuteCmdlet()
 
             try
             {
-                WriteWarningWithTimestamp("*** The namespace for all the model classes will change from Microsoft.Azure.Management.Monitor.Management.Models to Microsoft.Azure.Management.Monitor.Models in future releases.");
-                WriteWarningWithTimestamp("*** The namespace for output classes will be uniform for all classes in future releases to make it independent of modifications in the model classes.");
+                var isWarning =  bool.Parse(System.Environment.GetEnvironmentVariable("SuppressAzurePowerShellBreakingChangeWarnings"));
```

Comment:
shouldSuppress is more accurate.

Reply:
What will happen if Environment is not set?

Reply:
Test case is failed due to above reason.

Reply:
Got it,thanks